pulsar/backoff/backoff.go

```diff
@@ -38,6 +45,13 @@ type DefaultBackoff struct {
 	backoff time.Duration
 }
 
+func NewDefaultBackoff() Policy {
+	return &DefaultBackoff{}
+}
+func NewDefaultBackoffWithBackOff(backoff time.Duration) Policy {
```

Comment:
```suggestion
func NewDefaultBackoffWithInitialBackOff(backoff time.Duration) Policy {
```

Seems this name would be better?